CSS/main.css

```diff
@@ -0,0 +1,161 @@
+.body {
+  font-family: "Lato", sans-serif;
+}
+.page_header {
+  background: linear-gradient(
+    90deg,
+    rgba(213, 32, 71, 0.8) 0%,
+    rgba(238, 61, 27, 0.8) 100%
+  );
+  color: #ffffff;
+}
+.logo-img {
+  line-height: 29px;
+}
+.main-nav {
+  color: #ffffff;
+}
+.ul-nav {
+  list-style-type: none;
+}
+.li1 {
+  color: #ffffff;
+  font-size: 14px;
+  line-height: 17px;
+}
+.login_button {
+  background-color: unset;
+  color: #ffffff;
+  border-color: #ffffff;
+  font-size: 14px;
+  line-height: 17px;
+  border-radius: 6px;
+}
+.main_head {
+  font-style: normal;
+  font-size: 72px;
+  line-height: 86.4px;
+  color: #ffffff;
+  font-weight: bold;
+}
+.text_after_h1 {
+  font-style: normal;
+  font-size: 24px;
+  line-height: 36px;
+  color: #ffffff;
+}
+.sign_up_button {
+  background-color: #ffffff;
+  color: #ed4c5c;
+  border-color: unset;
+  border-radius: 6px;
+  font-size: 14px;
+  line-height: 16.8px;
+}
+.section_1 {
+  background-color: #ffffff;
+}
+.text_section_1 {
+  font-size: 16px;
+  line-height: 22.4px;
```

Comment:
The same. Check all the styles.

Reply:
I've checked out all styles and rounded them